instrumentation/log4j/log4j-appender-2.17/javaagent/build.gradle.kts

```diff
@@ -26,12 +26,62 @@ dependencies {
 }
 
 tasks {
+  test {
```

Comment:
Nit: WDYT about using the test suites feature instead?

Reply:
I think I found a good solution, ptal